src/pages/ContentScripts/features/developer-avator-openrank/index.tsx

```diff
@@ -0,0 +1,66 @@
+import features from '../../../../feature-manager';
+import { getOpenrank } from '../../../../api/developer';
+import elementReady from 'element-ready';
+
+const featureId = features.getFeatureID(import.meta.url);
+
+const getData = async (developerName: string): Promise<string | null> => {
+  const data = await getOpenrank(developerName);
+  if (data) {
+    const keys = Object.keys(data);
+    const latestKey = keys[keys.length - 1];
+    return data[latestKey];
+  }
+  return null;
+};
+
+const getDeveloperName = (target: HTMLElement): string | null => {
+  const hovercardUrl = target.getAttribute('data-hovercard-url');
```

Comment:
Let's change the naming. hovercardUrl -> hovercardUrlAttribute/hovercardUrlAttr since it is an attribute not a Url.

Reply:
get it